custom_components/garmin_connect/__init__.py

```
@@ -501,6 +504,19 @@ async def _async_update_data(self) -> dict:
             _LOGGER.debug("Error occurred while fetching Gear data: %s", err)
 
+        # Blood pressure data (isolated — users without a BP device may get 403/404)
+        try:
+            bp_data = await self.hass.async_add_executor_job(
+                self.api.get_blood_pressure, today.isoformat()
+            )
+            if bp_data:
+                _LOGGER.debug("Blood pressure data fetched: %s", bp_data)
+            else:
+                _LOGGER.debug("No blood pressure data found")
+        except Exception as bp_err:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)